x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CoreStart } from '@kbn/core/public';
+import {
+  ChromeNavigationNodeViewModel,
+  Navigation,
+  NavigationKibanaProvider,
+} from '@kbn/shared-ux-chrome-navigation';
+import React from 'react';
+
+// #TODO translate titles?
+const navItems: ChromeNavigationNodeViewModel[] = [
+  {
+    id: 'services-infra',
+    items: [
+      { id: 'services', title: 'Services', href: '/app/apm/services' },
+      {
+        id: 'infra',
+        title: 'Infrastructure',
+        href: '/app/metrics/inventory',
+      },
+    ],
+  },
+  {
+    id: 'alerts-cases-slos',
+    items: [
+      {
+        id: 'alerts',
+        title: 'Alerts',
+        href: '/app/observability/alerts',
+      },
+      {
+        id: 'Cases',
+        title: 'Cases',
+        href: '/app/observability/cases',
+      },
+      {
+        id: 'slos',
+        title: 'SLOs',
+        href: '/app/observability/slos',
+      },
+    ],
+  },
+  {
+    id: 'signals',
+    title: 'Signals',
+    items: [
+      {
+        id: 'traces',
+        title: 'Traces',
+        href: '/app/apm/traces',
+      },
+      {
+        id: 'logs',
+        title: 'Logs',
+        href: '/app/logs/stream',
+      },
+    ],
+  },
+  {
+    id: 'toolbox',
+    title: 'Toolbox',
+    items: [
+      {
+        id: 'visualization',
+        title: 'Visualization',
+        href: '/app/visualize',
+      },
+      {
+        id: 'dashboards',
+        title: 'Dashboards',
+        href: '/app/dashboards',
+      },
+    ],
+  },
+  {
+    id: 'on-boarding',
+    items: [
+      {
+        id: 'get-started',
+        title: 'Get started',
+        icon: 'launch',
+        href: '/app/observabilityOnboarding',
```

Comment:
@ogupte @yngrdyn 

Can I assume that `xpack.observability_onboarding.ui.enabled:` will be always true for serverless?

Reply:
Thanks. 🙏

Reply:
In case you are wondering if you should conditionally set items into this structure based on config, you shouldn't have to. Looking forward at the [next implementation of the Shared UX side navigation service](https://github.com/elastic/kibana/issues/156662), we will use "deep link" identifiers here, which can be validated against the chrome application service. If a plugin did not register a given deep link, we assume it is not enabled and remove it from the set of items before rendering.